Analysis/Tasks/PWGHF/HFCandidateCreator2Prong.cxx

```diff
@@ -107,7 +107,8 @@ struct HFCandidateCreator2Prong {
       auto errorDecayLengthXY = std::sqrt(getRotatedCovMatrixXX(covMatrixPV, phi, 0.) + getRotatedCovMatrixXX(covMatrixPCA, phi, 0.));
 
       // fill candidate table rows
-      rowCandidateBase(collision.posX(), collision.posY(), collision.posZ(),
+      rowCandidateBase(rowTrackIndexProng2.index0().collisionId(),
+                       collision.posX(), collision.posY(), collision.posZ(),
```

Comment:
Use the available collision row directly:
`rowTrackIndexProng2.index0().collisionId()` -> `collision.globalIndex()`